### ISSUE TITLE
Translation Request: German/Deutsch

### ISSUE THREAD

**Issue by PeterNerlich** [denysdovhan/bash-handbook#82]
I am able to translate this language! I don't know how much I would be able to translate in a given time span though because university, private projects and procrastination.

**Comment by denysdovhan** [comment]
It's okay to work on translation for a while. The important thing is to be in touch while you're translating. So if you're going to take a lead over German translation, please keep me updated, let me know what is your progress and how much you did.

If you wouldn't reply to my messages, then I will close this issue as not relevant.

**Comment by peternerlich** [comment]
Okay, I forked it and am translating over here: https://github.com/peternerlich/bash-handbook
Oh one thing, should I rather translate formal or informal? I currently think about translating formally and then 'translating' it again to informal in a different branch or file

**Comment by denysdovhan** [comment]
Translation process looks like this:

1. Fork the repo (you've already done with that).
2. Create a new folder in `translation`. Name it as the ISO-code of your language.
3. Copy `/README.md` to the folder and translate it.
4. Commit your changes and send a PR.
5. Find someone who can check your translation.
6. Fix mistakes in translation, commit and send the changes.
7. Wait for approval from the reviewer.
8. Profit!

**Comment by peternerlich** [comment]
I have currently stalled the translation process because I have some exams due atm. On the up side, this means I'll briefly have plenty of time once they're over!

**Comment by denysdovhan** [comment]
@PeterNerlich any updates?

**Comment by PeterNerlich** [comment]
Yes, admittedly I've had a hard time finding a fitting translation for `brace expansion`, but now that you asked, I really had to settle on something, whether I really like it or not. I'm not sure it's really recognized by that name in German, I didn't find any references I trusted in that respect. I'll probably be short on time a lot through this semester, however I do want to continue working on it, last but not least because I'm learning some tricks myself that way.
So in short, my progress since last time sadly is negligible, but there's no reason I'll settle on this new pace. Also sorry because I didn't drop a word for two months.

**Comment by denysdovhan** [comment]
Closing this because of no updates since April. Feel free to reopen, if you find more time.
